### DISCUSSION TITLE
Web based editor, pre-installed extensions

### DISCUSSION THREAD

**Discussion by eoinobrien** [community/community#13358] [Other Feature Feedback, Questions, & Ideas]
Is there a way to have a specific web extension pre-installed when the web-based editor is opened?

I've tried using the devcontainer.json but that didn't work, so I assume that is specific to pure codespaces.

I have a scenario where I'd like to create a [CodeTour](https://github.com/microsoft/codetour) for a code base and be able to have the end user open a simple web-based vscode editor to be able to navigate through the tour.

Does anyone know if it is possible to have the extension pre-installed in the repo?

**Comment by pedrocarvalho77** [comment]
Same problem here, @eoinobrien have you found any solution?

**Reply by eoinobrien** [reply]
I'm afraid I haven't, it wasn't vital to the project so have ignored it for now.
